src/applications/vaos/actions/newAppointment.js

```diff
@@ -546,36 +546,19 @@ export function getAppointmentSlots(startDate, endDate) {
               .endOf('month')
               .format('YYYY-MM-DD');
 
-        const response = await getAvailableSlots(
-          // Remove parse function when converting this call to FHIR service
-          parseFakeFHIRId(rootOrgId),
-          data.typeOfCareId,
-          data.clinicId,
-          startDateString,
-          endDateString,
-        );
-
-        const fetchedSlots = response[0]?.appointmentTimeSlot || [];
-        appointmentLength = response[0]?.appointmentLength;
+        const fetchedSlots = await getSlots({
+          siteId: rootOrgId,
+          typeOfCareId: data.typeOfCareId,
+          clinicId: data.clinicId,
+          startDate: startDateString,
+          endDate: endDateString,
+        });
 
         const now = moment();
 
-        mappedSlots = fetchedSlots.reduce((acc, slot) => {
-          /**
-           * The datetime we get back for startDateTime and endDateTime includes
```

Comment:
This code is covered in the transformer, where the bad offset is removed. I also took out the mapping to the object with a date/datetime, since it doesn't seem necessary. The calendar still leaves selected dates in that format. Might be worth refactoring this later, but for now I think this is ok, because it's mostly simpler and we want to be on a path toward using FHIR data structures as much as we can.